src/src/Globals/Plugins.cpp

```diff
@@ -243,24 +243,26 @@ void post_I2C_by_taskIndex(taskIndex_t taskIndex, deviceIndex_t DeviceIndex) {
 /**
  * Call the plugin of 1 task for 1 function, with standard EventStruct and optional command string
  */
-bool PluginCallForTask(taskIndex_t task, byte Function, EventStruct *TempEvent, String& command, bool unConditional = false) {
+bool PluginCallForTask(taskIndex_t taskIndex, byte Function, EventStruct *TempEvent, String& command, EventStruct *event = nullptr) {
```

Comment:
You have access to the TempEvent, but is there any reason why the `event` should not be a `const EventStruct*` ?

Reply:
This isn't declared in the .h file, should only be called from PluginCall

Reply:
I see.